cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -497,6 +499,26 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         })) {
           statsOnlyResultTable()
         }
+        p("The ability to modify the name of users can be granted via the `SET USER NAME` privilege. The following query shows an example of this:")
+        query("GRANT SET USER NAME ON DBMS TO statusModifier", ResultAssertions(r => {
```

Comment:
The SET DEFAULT DATABASE seems to be missing here as well, and there are other places that needs to be updated with both name and database, for example the note on line 524

Reply:
I just fixed places where I saw it blatantly missing, this PR is not about SET DEFAULT DATABASE at all though.

Reply:
I know it isn't but this note should be updated anyway: ``` p("Note that the combination of the `SET PASSWORDS` and the `SET USER STATUS` privilege actions is equivalent to the `ALTER USER` privilege action.")```

Maybe we should add a card about looking over if there are more places default database is missing?